tooling/nargo_fmt/tests/expected/constructor.nr

```diff
@@ -0,0 +1,8 @@
+struct Point {
+    x: u8,
+    y: u8,
+}
+
+fn foo() {
+    plain::Point { x: 5, y: 10 };
```

Comment:
This PR has added the plain:: prefix to the struct -- I don't think this should compile